internal/services/hdinsight/hdinsight_cluster_resource_test.go

```diff
@@ -0,0 +1,27 @@
+package hdinsight_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+)
+
+func TestAccHDInsightCluster(t *testing.T) {
```

Comment:
What's the reason for making these sequential?

Reply:
As you mentioned before, only one aadds is allowed per subs. So I make it sequence.